src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -1701,6 +1701,20 @@ public void GetPartsOfCastExpression(SyntaxNode node, out SyntaxNode type, out S
         public override SyntaxList<SyntaxNode> GetAttributeLists(SyntaxNode node)
             => node.GetAttributeLists();
 
+        public override bool IsParameterNameXmlElementSyntax(SyntaxNode node)
+            => node.IsKind(SyntaxKind.XmlElement) &&
+            ((XmlElementSyntax)node).StartTag.Name.ToString() == DocumentationCommentXmlNames.ParameterElementName;
+
+        public override SyntaxNode[] GetContentFromDocumentationCommentTriviaSyntax(SyntaxTrivia trivia)
+        {
+            if (trivia.GetStructure() is DocumentationCommentTriviaSyntax documentationCommentTrivia)
+            {
+                return documentationCommentTrivia.Content.ToArray();
```

Comment:
can we remve teh .ToArray?

Reply:
How?

Reply:
that can be casted to `SyntaxList<SyntaxNode>`

Reply:
Yip. Fixed. Thanks.